shared/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/hapi/HapiHelperTest.groovy

```diff
@@ -360,16 +360,15 @@ class HapiHelperTest extends Specification {
     }
 
     // PID-3.5 - Assigning Identifier Type Code
-    def "patient assigning authority methods work as expected"() {
+    def "patient assigning identifier methods work as expected"() {
         given:
-        def bundle = new Bundle()
         def pid3_5 = "pid3_5"
 
         when:
-        HapiFhirHelper.setPID3_5Coding(bundle, new Coding())
+        def bundle = new Bundle()
 
         then:
-        HapiHelper.getPID3_5Coding(bundle) == null
+        HapiFhirHelper.getPID3_5Value(bundle) == null
```

Comment:
If this is now just in `HapiFhirHelper`, can this test be removed?

Reply:
Removed the aasertion